docs/book/v6/tutorials/create-book-module.md

```diff
@@ -323,10 +230,12 @@ declare(strict_types=1);
 
 namespace Api\Book\Service;
 
-use Api\Book\Entity\Book;
-use Api\Book\Repository\BookRepository;
-use Dot\DependencyInjection\Attribute\Inject;
+use Core\App\Helper\Paginator;
+use Core\Book\Entity\Book;
+use Core\Book\Repository\BookRepository;
 use DateTimeImmutable;
+use Dot\DependencyInjection\Attribute\Inject;
+use Exception;
 
 class BookService implements BookServiceInterface
```

Comment:
This needs some refactoring, see [AdminService](https://github.com/dotkernel/api/blob/6.0/src/Admin/src/Service/AdminService.php) in `6.0` for example.
Then adapt `BookServiceInterface`.